src/SampleWeb/Query.cs

```diff
@@ -46,6 +48,28 @@ public class Query :
             name: "employeesConnection",
             resolve: context => context.DbContext.Employees);
 
+        #region NonNull test
+        Field<NonNullGraphType<CompanyGraph>>(
+            "companyNotNull",
+            arguments: new QueryArguments(
+                new QueryArgument<NonNullGraphType<IdGraphType>> { Name = "id" }
+            ),
+            resolve: context =>
+            {
+                var id = context.GetArgument<string>("id");
+                var dbContext = (GraphQlEfSampleDbContext)context.UserContext;
+                IQueryable<Company> query = dbContext.Companies;
+                var company = query.FirstOrDefault(x => x.Id == Int32.Parse(id));
+                if (company is null)
+                {
+                    throw new ExecutionError($"Company not found for id {id}");
```

Comment:
is a throw really necessary ? can it not be handled in a generic way, without every field needing to duplicate the same code?